source/standards/principle-least-access.html.md.erb

```diff
@@ -1,12 +1,12 @@
 ---
 title: Principle of Least Privilege
-last_reviewed_on: 2024-06-27
+last_reviewed_on: 2025-02-06
 review_in: 6 months
 ---
 
 # <%= current_page.data.title %>
 
-The [principle of least privilege][polp] involves setting up user accounts so they can only access and use the information they need for specific tasks. This can also apply to processes and individuals who might have to switch between normal access and the increased access of a superuser account as part of their work.
+The [principle of least privilege][polp] requires you to consider what permissions actors in your system need. For robotic or computer actors, like your deployment pipeline, this can be fairly easy to scope. Likewise, your end-users will have a clearly defined set of permissions they need to use your service.
```

Comment:
I feel like it would be good to keep the brief definition of the principle of least privilege.

Reply:
I don't know if it ever did, but the `polp` reference here doesn't actually seem to define the principle either... there's some [good discussion in NCSC guidance](https://www.ncsc.gov.uk/collection/zero-trust-architecture/know-your-user-service-device-identities#section_2) and it's worth noting that NCSC fairly consistently put the principle in quotes - my take is that's because this is almost a meta-principle: a guiding star to aim for rather than an inviolable policy with direct implications for designing a set of privileges.

Reply:
I don't think I'd agree with that. It's a guiding star and therefore it has direct implications for how we design things. 

To @lfdebrux's point - I agree, I'll add it back in